config/openshift/role.yaml

```diff
@@ -260,3 +260,21 @@ rules:
   - jobs
   verbs:
   - '*'
+- apiGroups:
+    - authentication.k8s.io
+  resources:
+    - tokenreviews
+  verbs:
+    - create
+- apiGroups:
+    - authorization.k8s.io
+  resources:
+    - subjectaccessreviews
+  verbs:
+    - create
+- apiGroups:
+    - results.tekton.dev
+  resources:
+    - '*'
+  verbs:
+    - '*'
```

Comment:
I see same changes at `config/base/role.yaml` 
So if changes are at base folder then we don't require it in Openshift folder :thinking: